en/engine/template/3.4-upgrade-to-3.6-guide.md

```diff
@@ -0,0 +1,85 @@
+# Project Engineering Upgrade Guide
+
+ - This article will introduce in detail the precautions for upgrading cocos creator 3D game from 3.4~3.4.x to 3.6. The modification is only for the 'native' directory under the project.
+
+### First, open the old project in the new directory with the new version of creator, then build the target platform, and finally refer to the following steps
+
+#### Android Java Modification Points
+  
+  - game/AppActivity.java
+  - game/InstantActivity.java
+    - ~~onCreate delete the following code~~
+    ```java
+          // Workaround in https://stackoverflow.com/questions/16283079/re-launch-of-activity-on-home-button-but-only-the-first-time/16447508
+          if (!isTaskRoot()) {
+              // Android launched another instance of the root activity into an existing task
+              //  so just quietly finish and go away, dropping the user back into the activity
+              //  at the top of the stack (ie: the last state of this task)
+              // Don't need to finish it again since it's finished in super.onCreate .
+              return;
+          }
+    ```
+
+  - app/AndroidManifest.xml
+    - delete code: ~~android:taskAffinity=""~~
+    - add code: android:exported="true"
+
+  - app/build.gradle 
+    - modify code: 
+    ``` html
+        "${RES_PATH}/assets" -> "${RES_PATH}/data"
+    ``` 
+
+#### cpp Modification Points
+  - remove file: ~~jni/main.cpp~~
+  - replace include file:  #include "cocos/platform/Application.h" —> #include "application/ApplicationManager.h"
+  - Change of usage: cc::Application::getInstance()->getScheduler() -> CC_CURRENT_ENGINE()->getScheduler()
```

Comment:
Uppercase for the first letter.
remove file -> Remove the file
replace include file -> Replace header file path